### ISSUE TITLE
Maximum Likelihood Estimation

### ISSUE THREAD

**Issue by AhmedThahir** [microsoft/LightGBM#6795]
Hi there!
I really love LightGBM's speed & performance all into one.

I want the model to output mean and variance, to be optimized using Maximum Likelihood Estimation. I'm guessing I would need to create a custom loss function, but I've not been able to wrap my head around it.

I want to create this question in the discussions forum, but couldn't find it, so I'm asking here.

Would really appreciate any help!
